synthea_omop_etl/dbt_project.yml

```diff
@@ -28,9 +28,9 @@ models:
 
 seeds:
   synthea_omop_etl:
-    synthea:
+    # synthea:
       +database: dbt_synthea
-      +schema: synthea
+      #+schema: synthea
```

Comment:
@adambouras before i close this, did you experience issues related to this config?